src/js/lib/client/src/index.js

```diff
@@ -172,7 +172,12 @@ optParser.options('crash-report-host', {
     describe    : 'The address of the server that will receive crash reports. ' +
                   'Consists of a hostname, optionally followed by a ":" and a port number',
     requiresArg : true,
-    default: cfg.defaultLogServerHost
+    default     : cfg.defaultLogServerHost
+})
+
+optParser.options('no-data-gathering', {
+    describe    : 'Disable the sharing of any usage data with the Enso developers',
```

Comment:
I would name it "Disable anonymous data collection" - it's shorter, a more popular name for that, and also doesn't mention "Enso" (we have a rule that we do not use product name too often if that's possible in order to keep the messages shorter + code easier to refactor).

Reply:
"Disable anonymous data collection" could be misleading, though. It could imply that non-anonymous data gathering is happening.

Reply:
I changed it to "Disable anonymous data collection". I assume it will be clear enough.

Reply:
@MichaelMauderer oh good point. @s9ferech please confirm it with @iamrecursion - she always knows how to write sentences that are not ambiguous (maybe this one is correct, just confirm it with Ara please)

Reply:
No. That one is definitely ambiguous. I would go with the one that is here originally.